LayoutTests/fast/css-grid-layout/simplified-layout-with-oof-children-crash.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<script>
+  if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+  }
+</script>
+<style>
+  .grid {
+    display: grid;
+    position: relative;
+    transform: translate(10px);
+  }
+  .abspos {
+    position: absolute;
+  }
+  .removeFromGrid {
+    grid-column: 1;
+  }
+</style>
+<body>
+  This test passes if it does not crash.
+    <div class="grid">
+      <div class="removeFromGrid abspos"></div>
+      <div class="removeFromGrid abspos"></div>
+      <div class="removeFromGrid abspos"></div>
+    </div>
+<script>
+let grid = document.querySelector(".grid");
+
+for (let i = 0; i < 100; ++i) {
+  let newElement = document.createElement("div");
+  newElement.classList += "abspos";
+  grid.appendChild(newElement);
+}
+document.body.offsetHeight;
+
+let elementsToRemove = Array.from(document.getElementsByClassName("removeFromGrid"));
+elementsToRemove.forEach(elementToRemove => {
+  grid.removeChild(elementToRemove);
+});
+document.body.offsetHeight;
+
+if (window.testRunner)
+  testRunner.notifyDone();
```

Comment:
this test does not look async. is it needed?

Reply:
nope